### PR TITLE
1060: Downgrade INFO logs to just that

### PR DESCRIPTION
**Pull request by gtmills**
BMCWEB_LOG_INFO vs BMCWEB_LOG_CRITICAL.

PE wants this backported to 1060.

Fixes 613773

### CODE REVIEW

**Comment by raviteja-b** [comment]
> BMCWEB_LOG_INFO vs BMCWEB_LOG_CRITICAL.
> 
> PE wants this backported to 1060.
> 
> Fixes 613773

there won't be debug info if we really hit a  Field  defect in these paths.

File: include/dump_offload.hpp [hunk 1]
```diff
@@ -188,8 +188,8 @@ class Handler : public std::enable_shared_from_this<Handler>
                 }
                 return;
             }
-            BMCWEB_LOG_CRITICAL << "INFO: Reset OffloadUri of " << dumpType
-                                << " dump id " << entryID;
+            BMCWEB_LOG_INFO << "INFO: Reset OffloadUri of " << dumpType
```
[the remark this answers is not in the document]
Comment (baemyung) [reply include/dump_offload.hpp:191]: - What if we set the default tracing as INFO? Will it cause too much of logs?

- Or what if using WARNING (and set the default level as WARNING too)?  I think WARNING seems better than CRITICAL.
https://github.com/openbmc/bmcweb/blob/fe37ddaf26abcf67e3861e2af3177333b09204f5/DEVELOPING.md?plain=1#L227

- For future, can we introduce another level for the unconditional logging?  e.g., BMCWEB_LOG_TRACE ?

[the remark this answers is not in the document]
Comment (baemyung) [reply include/dump_offload.hpp:191]: - What if we set the default tracing as INFO? Will it cause too much of logs?

- Or what if using WARNING (and set the default level as WARNING too)?  I think WARNING seems better than CRITICAL.
https://github.com/openbmc/bmcweb/blob/fe37ddaf26abcf67e3861e2af3177333b09204f5/DEVELOPING.md?plain=1#L227

- For future, can we introduce another level for the unconditional logging (e.g., BMCWEB_LOG_TRACE)?

[the remark this answers is not in the document]
Comment (gtmills) [reply include/dump_offload.hpp:191]: > What if we set the default tracing as INFO? Will it cause too much of logs?

Way too many. 
We would flood the journal.

[the remark this answers is not in the document]
Comment (gtmills) [reply include/dump_offload.hpp:191]: > For future, can we introduce another level for the unconditional logging (e.g., BMCWEB_LOG_TRACE)?

We discussed a LOG_IMPORTANT_INFO or something like that. Could sit between warning and error? 
WARNING might also have too many traces :/

**Comment by gtmills** [comment]
Slack discussion at https://ibm-systems-power.slack.com/archives/G3WAVQYTH/p1726246893628949?thread_ts=1726239559.931879&cid=G3WAVQYTH

File: include/dump_offload.hpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (gtmills) [reply include/dump_offload.hpp]: Discussed in slack, we are going to go with this to fix this defect.

**Comment by gtmills** [comment]
> there won't be debug info if we really hit a Field defect in these paths.

There will still be the audit log. Errors should be logged. These were only good path traces

**Comment by gtmills** [comment]
Discussed in slack, we are going to go with this to fix this defect.

File: redfish-core/lib/log_services.hpp [hunk 2]
```diff
@@ -1075,8 +1075,8 @@ inline void createDumpTaskCallback(
             }
             taskData->payload->httpHeaders.emplace_back(std::move(headerLoc));
 
-            BMCWEB_LOG_CRITICAL << "INFO: " << createdObjPath.str
-                                << ": Dump creation task completed";
+            BMCWEB_LOG_INFO << "INFO: " << createdObjPath.str
```
Comment (jeaaustx) [inline redfish-core/lib/log_services.hpp:1078]: Leaving "INFO:" in this message on purpose?

Comment (gtmills) [reply redfish-core/lib/log_services.hpp]: Done!

**Review by jeaaustx — APPROVED** [review]

**Review by baemyung — APPROVED** [review]
It looks good to me

**Pull request merged**